### ISSUE TITLE
Percentage font-size on :root causes wrong font-size in table with page/column breaks

### ISSUE THREAD

**Issue by MurakamiShinyu** [vivliostyle/vivliostyle.js#549]
Sample: https://gist.githack.com/MurakamiShinyu/84531775551580f0f2e1963f347430a9/raw/test.html

View with Vivliostyle: https://vivliostyle.github.io/vivliostyle.js/viewer/vivliostyle-viewer.html#b=https://gist.githack.com/MurakamiShinyu/84531775551580f0f2e1963f347430a9/raw/test.html

```css
:root {
  font-size: 150%;
}
```
```html
<html>
<body>
  <p>Body font size</p>
  <table>
    <tr><td>Table-cell font size</td></tr>
  </table>
  <hr>
  <table>
    <tr><td>Table-cell font size</td></tr>
    <tr><td>Table-cell font size</td></tr>
    ...
  </table>
</body>
</html>
```

The result font-size of "Body font size", the first "Table-cell font size" and the 2nd table's "Table-cell font size" must be same. However, the result font-size in the 2nd table becomes bigger when page breaks occur within the table.

This problem also happen with multi-column. Test [View with Vivliostyle](https://vivliostyle.github.io/vivliostyle.js/viewer/vivliostyle-viewer.html#b=https://gist.githack.com/MurakamiShinyu/84531775551580f0f2e1963f347430a9/raw/test.html&userStyle=data:,/*%3Cviewer%3E*/%0A/*%3C/viewer%3E*/%0A:root%20%7B%0A%20%20column-count:%202;%0A%7D) and the result:

![Screenshot 2019-08-29 17 34 39](https://user-images.githubusercontent.com/3324737/63925681-7ee66400-ca85-11e9-9c71-94d5b4e19ba7.png)

I confirmed that this bug existed since https://github.com/vivliostyle/vivliostyle.js/releases/tag/2017.2.

A workaround for this bug is specifying font-size on the table with unit other than `em` or `%`.
e.g. `table { font-size: 1rem; }`.

**Comment by MurakamiShinyu** [comment]
This bug seems to be a side effect of https://github.com/vivliostyle/vivliostyle.js/pull/311 (merged on 2016-11-30)

**Comment by MurakamiShinyu** [comment]
Fixed in #780 

Test: https://vivliostyle-git-fix-issue549-vivliostyle.vercel.app/#src=https://gist.githack.com/MurakamiShinyu/84531775551580f0f2e1963f347430a9/raw/test.html
